### DISCUSSION TITLE
Change to external LED's

### DISCUSSION THREAD

**Discussion by SargonofAssyria** [jomjol/AI-on-the-edge-device#3431] [General]
After using 15.7.0 for sometime without any problems, I got some errors about reading the right value.
I had some refection from the internal LED,and it seems that it created some errors on the 8 misread as 6.
I added 12 WS2812 LED's, it got a little bit better, but the picture changed to:
![alg_roi](https://github.com/user-attachments/assets/cf16a4f1-2681-498d-bc71-ef321e1257c2)
Those white/gray lines not there in the picture with the internal LED.
Is there a solution to get rid of those kind of moire lines and get a better light?

When I look at the live stream, picture quality is a little better and the lines are moving.

**Comment by caco3** [comment]
I never saw such effect.
Is your power supply sufficient?
Try to improve the contrast.

**Comment by SybexX** [comment]
````
Stripes in the picture

The horizontal stripes in the image, which are often criticized, are also related to a supply voltage that is too low. 
Here too, simply increasing the supply voltage at the 5 volt input of the ESP32 to 5.5 volts or more helps.
````
see https://nachbelichtet.com/esp32-cam-probleme-beheben
